SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/MasterMethods/CollectStatistics.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Plugin.Base;
+using Rpc.Service;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Common;
+
+namespace Plugin.Microsoft.Azure.SignalR.Benchmark.MasterMethods
+{
+    public class CollectStatistics : IMasterMethod
+    {
+        public Task Do(IDictionary<string, object> stepParameters, IDictionary<string, object> pluginParameters, IList<IRpcClient> clients)
+        {
+
+            Log.Information($"Start to collect statistics...");
+
+            // Get parameters
+            stepParameters.TryGetTypedValue(SignalRConstants.Interval, out double interval, Convert.ToDouble);
```

Comment:
Why interval's type is Double?

Reply:
Long/int is enough. The latency is not possible to be < 1ms when message go through network. I change the type of `interval` and `duration` to `long`, thanks.